apps/mediacenter/style/index.css

```diff
@@ -177,24 +203,47 @@ body {
   width: 24px;
 }
 
+.favorite.app img.icon {
+  top: unset;
+  bottom: 0.75em;
+  left: 1em;
+  width: 50px;
+  height: 50px;
+  border-radius: 40%;
+  background-color: #fff;
+  box-shadow: 0 3px 6px rgba(0,0,0,0.3);
+}
+
 .favorite .title {
   padding: 0.5em;
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
+  width: 100%;
+}
+
+.favorite.app .title {
+  padding-left: calc(1.25em + 50px);
```

Comment:
why this calc() ? Add a comment to explain.

Reply:
so i can get a quick perfect align of the text near the icon

Reply:
The comment should be in the CSS

Reply:
no need. i changed it to em instead